apps/server/src/modules/h5p-editor/controller/dto/h5p-file.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { Readable } from 'stream';
+import { File } from '@shared/infra/s3-client';
+
+export class H5pFileDto implements File {
+	constructor(file: H5pFileDto) {
+		this.name = file.name;
+		this.data = file.data;
+		this.mimeType = file.mimeType;
+	}
+
+	name: string;
+
+	data: Readable;
+
+	mimeType: string;
+}
+
+export interface GetH5pFileResponse {
+	data: Readable;
+	etag?: string;
+	contentType?: string;
+	contentLength?: number;
+	contentRange?: string;
+	name: string;
+}
+
+export interface GetLibraryFile {
```

Comment:
you can put this into the uc folder, ideally into a subfolder called dto like here

Reply:
8ab5968e98d35576a91f610fa0443370b27b6aea
Created a new dto subfolder inside uc and used it for h5p.uc.ts